pwndbg/commands/__init__.py

```diff
@@ -216,12 +216,17 @@ def split_args(self, argument):
 
 class ArgparsedCommand(object):
     """Adds documentation and offloads parsing for a Command via argparse"""
-    def __init__(self, parser):
-        self.parser = parser
+    def __init__(self, parser=None, desc=None):
+        assert parser is None or desc is None, 'Invalid ArgparsedCommand usage'
```

Comment:
I'd prefer a tiny bit of magic where we take a single argument and just check `isinstance(...)`